buildSrc/src/main/groovy/org.apache.groovy-common.gradle

```diff
@@ -42,10 +42,6 @@ group = 'org.apache.groovy'
 version = sharedConfiguration.groovyVersion.get()
 
 repositories {
-    if (rootProject.hasProperty('buildInChina')) {
-        // Try to use Aliyun maven repository when building in China
-        maven { url 'https://maven.aliyun.com/nexus/content/groups/public' }
-    }
```

Comment:
Aliyun maven shouldn't be applied here, if someone needed, can add this to `init.gradle`.